src/gg.regression.unity.bots/Tests/TestFramework/Scripts/RGTestUtils.cs

```diff
@@ -1,29 +1,36 @@
 using System;
 using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using RegressionGames.StateRecorder;
 using RegressionGames.StateRecorder.BotSegments;
 using RegressionGames.StateRecorder.BotSegments.Models;
+using RegressionGames.StateRecorder.Models;
 using RegressionGames.Types;
 using StateRecorder.BotSegments;
-using TMPro;
 using UnityEngine;
 using UnityEngine.Assertions;
 using UnityEngine.SceneManagement;
-using UnityEngine.UI;
 using Object = UnityEngine.Object;
 
+// ReSharper disable UnusedMember.Global - These are helper methods for customer projects, don't remove them
+// ReSharper disable MemberCanBePrivate.Global - These are helper methods for customer projects, don't make them private
 namespace RegressionGames.TestFramework
 {
-    /// <summary>
-    /// Utilities for running tests within the Unity Test Runner using Regression Games features.
-    /// </summary>
+    /**
+     * <summary>
+     * Utilities for running tests within the Unity Test Runner using Regression Games features.
+     * </summary>
+     */
     public class RGTestUtils
```

Comment:
Github folds this file by default, but this file is where the bulk of the meaningful changes are